tests/serializers/test_natural.py

```diff
@@ -67,6 +67,36 @@ def natural_key_test(format, self):
     self.assertIsNone(books[1].object.pk)
 
 
+class NaturalKeyWhenPkHasDefaultValueTestCase(TestCase):
+    """
+    Verify that deserializer does not ignore natural keys when primary key has a default value (ticket `28385`).
+
+    Test to prove that deserializer gets `pk` via `natural key` and updates object instead of create new one.
+    """
+
+    fixtures = ['fixture.json', 'fixture.xml']
+
+    def test_natural_key_json(self):
+        """
+        Test fixture based on `json` format.
+        """
+        management.call_command("loaddata", "fixture.json", verbosity=0)
+        self.assertEqual(Foo.objects.all().count(), 1)
```

Comment:
> Also, I think it's warranted to have an assert before the first import that checks for an empty table.

Why? Table by default is empty. If it isn't - `assert` (you suggest) won't passed. Better way is to clear table before first import. What do you think?

> Can you assert the values of the one object that exists in the db across imports, i.e. the pk didn't change.

Yes, I agree.